string mapname
int64 order
float64 precision
bool lossy
---
float64 allError 
float64 dynError 
int64 size
bool result
